db/schema.rb

```diff
@@ -40,7 +40,9 @@
     t.datetime "updated_at", null: false
     t.bigint "like_id"
     t.string "image"
+    t.bigint "user_id"
     t.index ["like_id"], name: "index_posts_on_like_id"
+    t.index ["user_id"], name: "index_posts_on_user_id"
   end
```

Comment:
They are! Indexes are different to columns. Worth researching for a spreadsheet aficionado such as yourself @adamjohnsnow!